hawk/src/test/java/com/orhanobut/hawk/HawkTest.java

```diff
@@ -44,7 +46,7 @@ public HawkTest() {
     context = Robolectric.buildActivity(Activity.class).create().get();
   }
 
-  @Override public void setUp() throws Exception {
+  @Before @Override public void setUp() throws Exception {
```

Comment:
is it something you want? the `setUp ` and `tearDown ` methods were never called